### PR TITLE
Add an alias for `Format` and `Quality`

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
Add an alias for `Format` and `Quality`

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [ ] Bug fix
- [x] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt [hunk 1]
```diff
@@ -13,6 +16,8 @@ class Transformation(internal val actions: List<Action> = emptyList()) : ITransf
     class Builder(private val components: MutableList<Action> = mutableListOf()) : ITransformable<Builder> {
         override fun add(action: Action) = apply { components.add(action) }
         fun build() = Transformation(components)
+        fun quality(quality: Quality) = apply { components.add(Delivery.quality(quality))}
```
Comment (const-cloudinary) [inline transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt]: Is there a better place to put these methods?

File: transformation-builder/src/test/kotlin/com/cloudinary/transformation/TransformationTest.kt [hunk 2]
```diff
@@ -349,4 +350,14 @@ class TransformationTest {
                 })
         })
     }
+
+    @Test
+    fun testQualityAlias() {
```
Comment (const-cloudinary) [inline transformation-builder/src/test/kotlin/com/cloudinary/transformation/TransformationTest.kt]: Maybe `testAliases`?
The test also includes `format`, which is not represented in the test name.

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: transformation-builder/src/test/kotlin/com/cloudinary/transformation/TransformationTest.kt (hunk 2 above)
Comment (adimiz1) [reply transformation-builder/src/test/kotlin/com/cloudinary/transformation/TransformationTest.kt]: Fixed

File: transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt (hunk 1 above)
Comment (adimiz1) [reply transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt]: As far as I understand we want them to be called on the transformation to shorten the syntax 
```
transformation {
    quality(Quality.auto())
}
```
Which will require the functions to be under the `Transformation` Builder class

File: transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt [hunk 3]
```diff
@@ -1,5 +1,8 @@
 package com.cloudinary.transformation
 
+import com.cloudinary.transformation.delivery.Delivery
```
Comment (const-cloudinary) [inline transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt]: do we need these imports here?

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt (hunk 3 above)
Comment (adimiz1) [reply transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt]: Removed

**Review by const-cloudinary — APPROVED** [review]

**Pull request merged**